### PR TITLE
Updated my patches to be up-to-date with 1.6

### PR DESCRIPTION
**Pull request by AluminumAlman**
Mods updated are:
- Black Widows;
- Thrumkin;
- Dragons! (Dragon Mod);
- Better Infestations;
- Cupro's Alloys;
- Vanilla Apparel Expanded;
- Vanilla Armour Expanded;
- Spartan Foundry;
- RoM - Bones;

i hope i didn't forget to patch something

### CODE REVIEW

File: Patches/Black Widows/ThingDefs_Misc/Patch_Apparel_Various.xml [hunk 1]
```diff
@@ -87,7 +93,13 @@
...
-					<ArmorRating_Sharp>0.55</ArmorRating_Sharp>
+					<ArmorRating_Sharp>14.4</ArmorRating_Sharp>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName = "Apparel_BWMAGISuitHelm"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Sharp>32.4</ArmorRating_Sharp>
 				</value>
 			</li>
```
Comment (LockdownX7) [inline Patches/Black Widows/ThingDefs_Misc/Patch_Apparel_Various.xml]: Both of these are setting ArmorRating_Sharp for the same def, the bottom one should be blunt.

File: Patches/Black Widows/ThingDefs_Weapons/Patch_Weapons.xml [hunk 2]
```diff
@@ -328,44 +325,47 @@
...
 				  </tools>
 				</value>
 			</li>
 			<li Class="PatchOperationAdd">
 				<xpath>Defs/ThingDef[defName="MeleeWeapon_bwscythe"]/equippedStatOffsets</xpath>
 				<value>
-					<MeleeCritChance>1</MeleeCritChance>
+					<MeleeCritChance>0.25</MeleeCritChance>
 					<MeleeParryChance>0.5</MeleeParryChance>
+					<MeleeDodgeChance>0.6</MeleeDodgeChance>
```
Comment (LockdownX7) [inline Patches/Black Widows/ThingDefs_Weapons/Patch_Weapons.xml]: Target def already has MeleeDodgeChance.

File: Patches/Black Widows/ThingDefs_Weapons/Patch_Weapons.xml [hunk 3]
```diff
@@ -54,7 +54,7 @@
 					  <graphicClass>Graphic_StackCount</graphicClass>
 					</graphicData>
 					<statBases>
-					  <MarketValue>0.52</MarketValue>
+					  <MarketValue>0.</MarketValue>
```
Comment (LockdownX7) [inline Patches/Black Widows/ThingDefs_Weapons/Patch_Weapons.xml]: Typo?

Comment (AluminumAlman) [reply Patches/Black Widows/ThingDefs_Weapons/Patch_Weapons.xml]: Fixed all of your pointed out issues with the latest commit.

**Pull request merged**